BibBuddy/app/src/main/res/values/strings.xml

```diff
@@ -49,12 +56,27 @@
     <string name="add_book">Buch hinzufügen</string>
     <string name="added_book">Buch wurde hinzufügt</string>
     <string name="book_icon">Buch Icon</string>
+    <string name="add_book_tooltip">Neues Buch hinzufügen</string>
+    <string name="book_help">Um ein neues Buches hinzuzufügen, drücken Sie den \"+\"-Button. Es
```

Comment:
The plus button is shown in the text as "/+/" - Button. I suggest changing the style to e.g. "+ Button" or "plus-Button" or even better using the picture of the plus-Button that can be found in the "Library" ("Bibliothek")-tab.

Reply:
It is changed in **+-Button**, because to add an image is difficult and in my opinion not necessary.